src/Playwright.Tests/BrowserContextStrictSelectorsTests.cs

```diff
@@ -0,0 +1,71 @@
+/*
+ * MIT License
+ *
+ * Copyright (c) 2020 Darío Kondratiuk
```

Comment:
in theory for all new files we can keep MS only copyright?

Reply:
yeah, copy-pasta error, good spot